.vscode/cspell.json

```
@@ -28,14 +30,17 @@
     "otlp",
     "OTLP",
```

"OTEP" above should be "otep", with an entry added to textlint. Ditto for OTLP:

```suggestion
    "otlp",
```